src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,186 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> node;
+    private Node<T> head = null;
```

Comment:
Fields in class implements automatically.
If it link, it's null.
You can deletedn '= null'.